### DISCUSSION TITLE
What is supported for multiplayer on dedicated servers?

### DISCUSSION THREAD

**Discussion by mdano01** [Courseplay/Courseplay_FS22#2713] [Q&A] [answered]
So we have course play working on our server but I noticed that when a player saves a course it's only saved for that person. The rest of us can not see it. While looking around for answers I never found anything official that says exactly what multiplayer support for covers. I assume by my findings that courses are only saved on the players local machine that creates them. Has anybody found a way to sync the saved courses either to the server or other players. I had some ideas like saving them to a shared google folder but wanted to see if there was any officially supported way or if everyone has just lived without this ability. Any ideas would be welcome, since we love the mod and everything it does.

**Comment by Tensuko** [comment, marked as answer]
Courses are only saved localy.
When you want to share the course, share the files or load them in a vehicle and the other can save it.
